instrumentation/opentelemetry-instrumentation-django/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - Django instrumentation is now enabled by default but can be disabled by setting `OTEL_PYTHON_DJANGO_INSTRUMENT` to `False` ([#1239](https://github.com/open-telemetry/opentelemetry-python/pull/1239))
+- Bugfix use request.path replace request.get_full_path(). It will get correct span name ([#1309](https://github.com/open-telemetry/opentelemetry-python/pull/1309#))
```

Comment:
Typically with changelogs, you should try to have information that's relevant to the user, so mainly behavior.

In this case, it's sufficient to say "span name resolves correctly for paths that also require a query parameter to match".

Reply:
Thank you for your suggestion, I will pay attention to it later